docs/6.x/pack.md

```diff
@@ -912,3 +912,32 @@ When building a Cluster Image, `tele build` will discover `custom-planet:1.0.0`
 Docker image and vendor it along with other dependencies. During the Cluster
 installation all nodes with the role `worker` will use the custom "planet" Docker
 image instead of the default one.
+
+## Support For Intermediate Upgrades
```

Comment:
This is more of a nitpick but I think that "intermediate upgrades" term we coined here is kinda confusing :) Maybe say something like "Direct Upgrades From Older LTS Versions"? I.e. so a user reading the title (or searching the docs) would immediately understand what this is about. Maybe it's just me though.